app/js/components/common/Header.js

```diff
@@ -112,7 +112,7 @@ export class Header extends Component {
 
                 <Link to="" activeClassName="active">
                     <li>
-                        <a href="#">Logout {' '}
+                        <a href="/openmrs/appui/header/logout.action?successUrl=openmrs">Logout {' '}
```

Comment:
Why do you want to avoid creating a new ticket for something totally different from what this pull request was all about?